compiler/src/iree/compiler/Codegen/Common/test/iree_loop_invariant_code_motion.mlir

```diff
@@ -0,0 +1,95 @@
+// RUN: iree-opt %s  -split-input-file --iree-loop-invariant-code-motion | FileCheck %s
```

Comment:
nit: We don't need splits for in test

Reply:
I prefer the splits because if I add prints or `--mlir-print-ir-after`/`--debug` dumps it will keep those prints within the split regions because it runs the command once per split rather than all at once.